patches/api/0085-add-PlayerAdvancementCriterionGrantEvent.patch

```diff
@@ -1,15 +1,15 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: BillyGalbreath <Blake.Galbreath@GMail.com>
 Date: Fri, 19 Jan 2018 08:15:14 -0600
-Subject: [PATCH] PlayerAdvancementCriterionGrantEvent
+Subject: [PATCH] add PlayerAdvancementCriterionGrantEvent
```

Comment:
```suggestion
Subject: [PATCH] Add PlayerAdvancementCriterionGrantEvent
```
Renaming this isn't really required but if you would like to go through with it then i think you should make it start with an uppercase letter since 99 % of patches start with one